src/main/java/ai/elimu/model/contributor/StoryBookPeerReviewEvent.java

```diff
@@ -68,6 +68,7 @@ public void setApproved(Boolean isApproved) {
         this.approved = isApproved;
     }
 
+    @Column(length = 1000)
```

Comment:
```suggestion
```

The @Column annotation is not for get methods.

Reply:
Done